### PR TITLE
Document prometheus-meta-operator's storage decisions

### PR DESCRIPTION
**Pull request by omegas27**

### CODE REVIEW

File: Documentation/storage.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Retention
+
+Max Retention duration: **2 weeks**
+
+Max Retention size: **95 GB**
+
+## Volume sizing
+
+Previous Volume Size: **20 GB**, this was not enough as some prometheus needs more
```
Comment (JosephSalisbury) [inline Documentation/storage.md:19]: I don't think we need to document this

File: Documentation/storage.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Current Volume Size: **100GB**
+
+Reasons: https://gigantic.slack.com/archives/C01176DKNP4/p1601471521029900
```
Comment (JosephSalisbury) [inline Documentation/storage.md:23]: Do we want to copy the pertinent stuff out of slack?

**Comment by omegas27** [comment]
I took most of it out :)

**Review by JosephSalisbury — APPROVED** [review]
Let's scrub the installation names (i don't _love_ them being in public repos), but lgtm

**Pull request merged**